lib.js

```diff
@@ -6,9 +6,12 @@ const ms = require('ms');
 const request = require('request-promise');
 const xtend = require('xtend');
 const pki = require('node-forge').pki;
+const moment = require('moment');
 const path = require('path');
 const { URL } = require('url');
 
+const EXPIRATION_IN_DAYS = 30;
```

Comment:
Can be repliaced by `const EXPIRATION = ms('30 days')` if we get rid of the moment dependency.

Reply:
removed the dependency